docs/source/configuration/telemetry_next/metrics/datadog.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Collecting metrics in the Apollo Router
+---
+
+## Datadog via OTLP
+
+To use Datadog, you must configure the Datadog agent to accept OTLP metrics. This can be done by adding the following to your `datadog.yaml`:
+
+```yaml title="datadog.yaml"
+otlp_config:
+  receiver:
+    protocols:
+      grpc:
+        endpoint: <dd-agent-ip>:4317
+```
+
+The router must also be configured to send traces to the Datadog agent:
+
+*Ensure that you set `temporality` to `delta`. Failure to do this will result in incorrect metrics.*
```

Comment:
Use the new admonition component? https://github.com/apollographql/docs/#admonitions